src/js/Cindy.js.wrapper

```diff
@@ -1,5 +1,7 @@
 /* CindyJS - (C) 2014-2015  J. Richter-Gebert
```

Comment:
I didn't modify that line, but I assume you are right. Will augment my commit in this regard.